pkg/pubsub/adapter/converters/build.go

```diff
@@ -27,7 +27,7 @@ import (
 )
 
 const (
-	buildSchemaUrl = "https://raw.githubusercontent.com/google/knative-gcp/master/schemas/build/schema.json"
+	buildSchemaUrl = "https://raw.githubusercontent.com/googleapis/google-cloudevents/master/proto/google/events/cloud/build/v1/data.proto"
```

Comment:
This will change to `https://raw.githubusercontent.com/googleapis/google-cloudevents/master/proto/google/events/cloud/cloudbuild/v1/data.proto` because of this PR googleapis/google-cloudevents#57.